buildah-fedora/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.fedoraproject.org/fedora:27
+
+ENV VERSION=0 RELEASE=1 ARCH=x86_64
+LABEL com.redhat.component="buildah" \
+      atomic.run="once" \
+      atomic.type="system"
+
+RUN mkdir /exports /host && dnf install --setopt=tsflags=nodocs -y buildah && dnf clean all
```

Comment:
nit: It's always nice to have commands on different lines.

Reply:
thanks, I've pushed a new version to split this on multiple lines